osmosis-1/osmosis-frontier.assetlist.json

```diff
@@ -1517,6 +1517,36 @@
       },
       "coingecko_id": "dai"
     },
+    {
+      "description": "Gravity Bridge BTC",
+      "denom_units": [
+        {
+          "denom": "ibc/C9B0D48FD2C5B91135F118FF2484551888966590D7BDC20F6A87308DBA670796",
+          "exponent": 0,
+          "aliases": [
+            "satoshi"
+          ]
+        },
+        {
+          "denom": "btc",
+          "exponent": 8
```

Comment:
after switching to wtbc, is exponent 8 still accurate?

Reply:
Yes, thankfully they set the exponent correctly when deploying the ERC20 contract: https://etherscan.io/token/0x2260fac5e5542a773aa44fbcfedf7c193bc2c599